atlassian_jwt_auth/key.py

```diff
@@ -1,21 +1,24 @@
 import base64
 import cgi
+import logging
 import os
 import re
 import sys
 
 import cachecontrol
 import cryptography.hazmat.backends
-from cryptography.hazmat.primitives import serialization
 import jwt
 import requests
+from cryptography.hazmat.primitives import serialization
+from requests.exceptions import RequestException
 
 if sys.version_info[0] >= 3:
     from urllib.parse import unquote_plus
 else:
     from urllib import unquote_plus
 
 PEM_FILE_TYPE = 'application/x-pem-file'
+logger = logging.getLogger(__name__)
```

Comment:
I prefer to avoid globals such as this. Perhaps make a `_get_logger()` method  or obtain a logger where you need it.